### PR TITLE
Extend COCO format to skeletons

### PR DESCRIPTION
**Pull request by yasakova-anastasia**
### Motivation and context

- Added `COCO Keypoints 1.0` for skeleton import/export

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [x] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: cvat/apps/dataset_manager/bindings.py [hunk 1]
```diff
@@ -1056,14 +1059,24 @@ def _load_categories(labels: list):
             datum_annotation.Categories] = {}
 
         label_categories = datum_annotation.LabelCategories(attributes=['occluded'])
+        point_categories = datum_annotation.PointsCategories()
 
         for _, label in labels:
-            label_categories.add(label['name'])
-            for _, attr in label['attributes']:
-                label_categories.attributes.add(attr['name'])
+            if label.get('parent') is None:
```
Comment (zhiltsov-max) [inline cvat/apps/dataset_manager/bindings.py:1065]: 1. The opposite case may be missed, please check if it needs to be covered too.
2. Probably, metadata in projects and tasks is different, please check if something needs to be changed about this.

File: cvat/apps/dataset_manager/bindings.py [hunk 2]
```diff
@@ -1056,14 +1059,24 @@ def _load_categories(labels: list):
...
+                label_id = label_categories.add(label['name'])
+                for _, attr in label['attributes']:
+                    label_categories.attributes.add(attr['name'])
+
+                if label['type'] == str(LabelType.SKELETON):
+                    labels_from = [int(l[16:-1]) for l in re.findall(f'data-node-from="\\w*"', label['svg'])]
+                    labels_to = [int(l[14:-1]) for l in re.findall(f'data-node-to="\\w*"', label['svg'])]
+                    sublabels = [l[17:-1] for l in re.findall(f'data-label-name="\\w*"', label['svg'])]
```
Comment (zhiltsov-max) [inline cvat/apps/dataset_manager/bindings.py:1073]: I'd suggest to use regex capture groups and raw strings here to simplify code:

```python
digits = re.match(r'data-node-from="(\d+)"')
```

File: cvat/apps/dataset_manager/bindings.py [hunk 3]
```diff
@@ -1056,14 +1059,24 @@ def _load_categories(labels: list):
...
+                    joints = [[from_, to_] for from_, to_ in zip(labels_from, labels_to)]
```
Comment (zhiltsov-max) [inline cvat/apps/dataset_manager/bindings.py:1074]:
```python
                    joints = [list(v) for v in zip(labels_from, labels_to)]
```
or
```python
                    joints = list(map(list, zip(labels_from, labels_to)))
```

**Comment by zhiltsov-max** [comment]
1. When I try to export a project with skeletons in COCO, the export fails (`KeyError: <number>`)
2. @bsekachev, when we go to project/task label config, we can't dump skeletons after project/task is created. I think, it should be available. The dump button is grey and inactive.
Screenshot from 2022-08-22 16-40-25

**Comment by bsekachev** [comment]
@zhiltsov-max 

> @bsekachev, when we go to project/task label config, we can't dump skeletons after project/task is created. I think, it should be available. The dump button is grey and inactive.

Fixed here https://github.com/opencv/cvat/pull/4829

**Pull request closed without merging**